applications/example/config/passport.js

```diff
@@ -1,40 +0,0 @@
-if (api.passport) {
```

Comment:
But the rationale of this change should be described in the full commit message.